src/components/Login/Login.js

```diff
@@ -1,8 +1,118 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
 
-const Login = () => {
-  return <div>Login</div>;
+const Login = ({ setModalOpen }) => {
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  const [showPw, setShowPw] = useState(false);
+  const [userInfo, setUserInfo] = useState({ email: '', password: '' });
+
+  function getUserInfo(e) {
+    const { name, value } = e.target;
+    setUserInfo({ ...userInfo, [name]: value });
+  }
+  console.log(userInfo);
+  //이메일 밑에 p 태그
+  const emailChk =
+    (userInfo.email.includes('@') && userInfo.email.includes('.')) ||
+    !userInfo.email;
+  // console.log(emailChk);
+  //비밀번호 밑에 p 태그
+  const pwChk = userInfo.password.length > 5 || !userInfo.password;
+
+  //링크 넘어가게
+  const isUserVaidate =
+    userInfo.email.includes('@') &&
+    userInfo.email.includes('.') &&
+    userInfo.password.length >= 6;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/');
+  };
+  function validateUser(e) {
+    e.preventDefault();
+
+    fetch('http://10.58.52.88:8002/users/login', {
+      method: 'POST',
+      headers: { 'Content-type': 'application/json;charset=utf-8' },
+      body: JSON.stringify(userInfo),
+    })
+      .then(response => response.json())
+
+      .then(data => {
+        if (data.message === 'login success') {
+          navigate({ goToMain });
+        }
```

Comment:
```suggestion
        if (data.message === 'login success') {
          navigate('/');
        } else {
          alert("다시 로그인을 해주세요!")
        }
```

메인 페이지로 이동하려면 함수를 실행하는 것이 아닌 위와 같은 방법으로 구현해주시고
로그인이 성공했을때 말고 통신에 실패했을 때도 분기 처리 해주는 것이 좋은 사용자의 경험에 이어집니다!